js/viewmodel.js

```diff
@@ -44,7 +44,7 @@ window.FBE = window.FBE || {};
 
     function loadEntity(placeables, entity) {
         var items = placeables
-            .filter(function (p) { return p.name === entity.name; }),
+                .filter(function (p) { return p.name === entity.name; }),

```

Comment:
can we move this onto the previous line, or give `item` there it's own `var` statement.